webapp/store/views.py

```diff
@@ -274,6 +276,21 @@ def brand_search_snap():
             status_code,
         )
 
+
+    @store.route("/youtube/<video_id>")
+    def get_video_thumbnail_data(video_id):
+        API_KEY = "foo"
```

Comment:
Here you can do what I told you on mattermost:
```suggestion
        API_KEY = os.getenv("API_KEY_NAME_FOR_THIS_SERVICE")
```

And then you can use `env.local` to store the value. This line will be better outside the function, at the top of the file line ~24